tools/kotlin/BUILD.bazel

```diff
@@ -2,15 +2,15 @@
 Configurations for a codebase-wide build toolchain for Kotlin.
 """
 
-load("@io_bazel_rules_kotlin//kotlin:core.bzl", "define_kt_toolchain", "kt_javac_options", "kt_kotlinc_options")
+load("@io_bazel_rules_kotlin//kotlin:core.bzl", "define_kt_toolchain", "kt_compiler_plugin", "kt_javac_options", "kt_kotlinc_options")
 
 # This exposes a patch that fixes an issue with duplicate processor arguments. See:
 # github.com/bazelbuild/rules_kotlin/pull/940.
 exports_files(["remove_processor_duplicates.patch"])
 
 kt_kotlinc_options(
     name = "oppia_kotlinc_options",
-    warn = "error",
+    warn = "report",
```

Comment:
@BenHenning, is it intentional changing the WError to report here?

Reply:
As discussed in the CLaM meeting, this was just to simplify this prototype PR though I've made a note in the Kotlin upgrade PR to make this a bit more lenient for local development (but still strict for CI so that we can continue to keep build warnings at a minimum).